### ISSUE TITLE
[Bug] Using a custom Button renderer on Android results in Talkback requiring an extra swipe to move accessibility focus

### ISSUE THREAD

**Issue by johnshardman** [xamarin/Xamarin.Forms#13075]
Where custom renderers are used for Buttons on Android, with renderers derived from Xamarin.Forms.Platform.Android.AppCompat.ButtonRenderer, users who use Talkback have to do an additional swipe to move the accessibility focus from one Button to another.

This can be reproduced by adding a custom renderer that derives from Xamarin.Forms.Platform.Android.AppCompat.ButtonRenderer but that does not have any additional code in it. Run the app, enable Talkback, tap on the Button and then swipe sideways. It then takes another swipe to actually move the focus, which should not be necessary.

### Basic Information

- Version with issue:
- Last known good version: XF 4.8.0.1451
- Platform Target Frameworks:
  - Android: 9.0
- Android Support Library / AndroidX Version: 28.0.0.3

**Comment by PureWeen** [comment]
@johnshardman if you inherit from 
`Xamarin.Forms.Platform.Android.FastRenderers.ButtonRenderer` instead do you still see the same issue?

**Comment by johnshardman** [comment]
@PureWeen - Inheriting from FastRenderers.ButtonRenderer does not exhibit the problem. Interesting - is it possible that the ViewGroup is in the accessibility tree when inheriting from AppCompat.ButtonRenderer ?

**Comment by PureWeen** [comment]
This issue should be resolved by this PR  @johnshardman 

https://github.com/xamarin/Xamarin.Forms/pull/13244
